### PR TITLE
Key travel information

### PR DESCRIPTION
**Pull request by mirhamasala**
- Create function `findTravelnfoForCountry`
- Create component `TravelInfoText`
- Create component `TravelInfo`
- Add `KeyTravelInfo` to `Country` page
- Extract travel criteria into JSON
- Add Key Info link to `ContentNav`

Screenshot 2020-09-01 at 5 25 04 PM

### CODE REVIEW

File: src/components/Country/TravelInfo.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <section>
+      <p>Are there any quarantine requirements upon arrival?</p>
```
Comment (yurikoval) [inline src/components/Country/TravelInfo.vue:4]: Would be nice to have this header in the same places as the answers.

**Review by yurikoval — COMMENTED** [review]
I think this is good -- another approach would be to have a separate file that is not a component and export the json for readability and maintainability. Then import it in the `TravelInfo` model. Also, it would be nice to see how you would handle missing keys in the json -- `unknown`

File: src/components/Country/TravelInfo.vue [hunk 2]
```diff
@@ -0,0 +1,141 @@
+<template>
+  <div>
+    <TravelInfoText :country="country.name" criteria="inbound"
```
Comment (yurikoval) [inline src/components/Country/TravelInfo.vue:3]: There is a lot of repetition here -- can you try to refactor this further?

**Review by yurikoval — DISMISSED** [review]

**Comment by jannwong** [comment]
@yurikoval @mirhamasala when a key has no value, we should not display the question and the answer. Currently it's displaying the question and answer is unknown

**Comment by mirhamasala** [comment]
> @yurikoval @mirhamasala when a key has no value, we should not display the question and the answer. Currently it's displaying the question and answer is unknown

@jannwong Done

**Review by yurikoval — APPROVED** [review]

**Pull request merged**